web/src/engine/websites/Digimon.ts

```diff
@@ -0,0 +1,56 @@
+import { Tags } from '../Tags';
+import icon from './Digimon.webp';
+import { type Chapter, DecoratableMangaScraper, Manga, type MangaPlugin } from '../providers/MangaPlugin';
+import * as Common from './decorators/Common';
+import { FetchCSS } from '../platform/FetchProvider';
+
+function ChapterScript(mangaId: string): string {
+    return `new Promise(resolve => {
+        resolve([...document.querySelectorAll('div[id="${mangaId}"] li.pg-ep_list__item a')].map(chapter => {
+            return {
+                id: new URL(chapter.pathname, window.location).pathname,
+                title: chapter.text.trim()
+            }
+        }).filter(element => element.id != window.location.pathname));
+    });
+`;
+}
+
+@Common.PagesSinglePageJS(`[...document.querySelectorAll('div.swiper ul li[aria-label] > img')].reverse().map(image => new URL(image.src, window.location).href);`, 500)
+@Common.ImageAjax()
+export default class extends DecoratableMangaScraper {
```

Comment:
There are only a handful of mangas/chapters, is it really worth to add support for this website?

Reply:
We have websites with less mangas (for particular reasons ), and more chapters will come i guess🤷